.docker/docker-compose.yml

```diff
@@ -4,6 +4,8 @@ services:
   app:
     build:
       context: ../
+    environment:
+      DATABASE_CONNECTION_STRING: postgres://postgres:postgres@db:5432/articles
```

Comment:
Reasonable to pass in environment variables for configuration. However, separating the components makes it easier to configure them as they could be managed differently:

- port hardcoded in a Helm chart (always `5432`)
- host generate by a Helm chart (e.g. `article-store--test--postgres`)
- password coming from a Kubernetes `Secret`